packages/@aws-cdk/aws-certificatemanager/lambda-packages/dns_validated_certificate_handler/lib/index.js

```diff
@@ -226,7 +227,7 @@ exports.certificateRequestHandler = async function(event, context) {
           event.ResourceProperties.DomainName,
           event.ResourceProperties.SubjectAlternativeNames,
           event.ResourceProperties.HostedZoneId,
-          event.ResourceProperties.Region,
+          event.ResourceProperties.Region
```

Comment:
Please do not remove this trailing comma.

Reply:
@skinny85 I did not! The eslint config in the package did!